webui/tests/cypress/integration/02-loginout.spec.ts

```diff
@@ -46,11 +46,16 @@ describe('Sign in/out', () => {
       .type(username, { delay: 100 })
       .should('have.value', username);
 
-    cy.server();
-    cy.route('POST', /\/login.*/).as('loginRequest');
-    cy.get('button[type="submit"]').click();
-    cy.wait('@loginRequest');
-    cy.checkLoggedIn(username);
+    cy.get('button[type="submit"]').contains('Sign In').click();
+
+    /*
+     * Cypress is unable to capture /api/v1/auth/login POST requests properly
+     * via `cy.route`, instead having to rely on a time-based wait.
+     * https://github.com/cypress-io/cypress/issues/2188
+     */
+    /* eslint-disable-next-line cypress/no-unnecessary-waiting */
+    cy.wait(500);
```

Comment:
thoughts on setting this higher? 1s? we could have this as a constant and potentially share it.

Reply:
extracted into it's own constant as well: `DEFAULT_WAIT_TIME`